src/v2/view-builder/views/PollView.js

```diff
@@ -0,0 +1,54 @@
+import { loc, View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+import polling from './shared/polling';
+import { POLL_REQUEST_INTERVAL } from '../utils/Constants';
+
+const PollMessageView = View.extend({
+  template: hbs`
+    <div class="ion-messages-container">
+      {{i18n code="poll.form.message" bundle="login" arguments="interval" $1="<span class='strong'>$1</span>"}}
+    </div>
+    <div class="okta-waiting-spinner"></div>`
+  ,
+  getTemplateData () {
+    const interval = this.options;
+    return {
+      interval,
+    };
+  },
+});
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    title () {
+      return  loc('poll.form.title', 'login');
+    },
+
+    noButtonBar: true,
+    
+    initialize () {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      this.startPolling();
+    },
+
+    render () {
+      BaseForm.prototype.render.apply(this, arguments);
+      this.pollingInterval = Math.ceil(this.options.appState.getCurrentViewState().refresh / 1000);
```

Comment:
What are the units for `refresh` and `pollingInterval`?  Ultimately this is for `setInterval` so `pollingInterval` should be milliseconds? But `refresh` from the mock looks like it's already in ms? 
It would be easier to reason what the calculation is doing if the constant 1000 was a variable like `MS_PER_SEC` and `pollingInterval` included its units, `pollingIntervalMS`.

Reply:
I think i should change the variable name.. it's not for setInterval, it's a countDown counter.. for if `refresh:5000` then enduser would see `5 seconds` `4seconds`.... and so on. See the screenshot.

I will change the variable name.

Reply:
To clear up possible future confusion, there should be an explanation of what's going on here. I think this would do it?
```
// Display countdown to next poll in seconds, convert refresh (ms) to seconds
const pollingIntervalSeconds = Math.ceil(this.options.appState.getCurrentViewState().refresh / MS_PER_S);
this.pollingInterval = pollingIntervalSeconds;
```
and declare the constant outside of this method so it can be re-used each time render is called.

Reply:
I changed the variable name to make it more clear what it does..